src/rpc/mining.cpp

```diff
@@ -200,7 +217,7 @@ UniValue generate(const UniValue& params, bool fHelp)
     if (coinbaseScript->reserveScript.empty())
         throw JSONRPCError(RPC_INTERNAL_ERROR, "No coinbase script available (mining requires a wallet)");
 
-    return generateBlocks(coinbaseScript, nGenerate, nMaxTries, true);
+    return generateBlocks(coinbaseScript, nGenerate, nMaxTries, true, generateWeak);
```

Comment:
do you want to convert this to a bool before passing in?

Reply:
Well, rather the enum as above, but yeah. Making it a self-describing string argument is probably a good idea as well?